ts/chewing-editor_zh_TW.ts

```diff
@@ -179,7 +179,7 @@
     <message>
         <location filename="../src/view/Notification.cpp" line="94"/>
         <source>Undo completed. Readd %1.</source>
-        <translation>回覆完成。重新加入 「%1」。</translation>
+        <translation>復原完成。重新加入 「%1」。</translation>
```

Comment:
Or just add `--amend` with `git commit` so that you don't need to rebase/squash it.